build.gradle

```diff
@@ -18,32 +18,23 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
-
-/**
-* The application plugin facilitates creating an executable JVM application.
-*/
+ 
```

Comment:
This trailing white space needs to be removed.

Reply:
Finished & pushed